packages/client/src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,298 @@
+              fontFamily: "'Press Start 2P', cursive",
+            }}
+          >
+            {isAuth ? "Играть" : "Войти"}
+          </Link>
+          <Link
+            variant="h4"
+            href="#about"
+            color="#FFC08B"
+            sx={{
+              textAlign: "center",
+              marginBottom: "60px",
+              cursor: "pointer",
+              color: "#FFC08B",
+              fontFamily: "'Press Start 2P', cursive",
+            }}
+            onClick={() => scrollToElement("about")}
+          >
+            Об игре
+          </Link>
+          <Link
+            variant="h4"
+            component={RouterLink}
+            to="/leaderboard"
+            color="#FFC08B"
+            sx={{
+              textAlign: "center",
+              marginBottom: "60px",
+              cursor: "pointer",
+              color: "#FFC08B",
+              fontFamily: "'Press Start 2P', cursive",
+            }}
+          >
+            Лидерборд
+          </Link>
+          <Box
+            mt={5}
+            sx={{
+              backgroundImage: "url(../../../public/images/water-tiles.gif)",
+              backgroundRepeat: "no-repeat",
+              backgroundSize: "cover",
+              width: "256px",
+              height: "256px",
+              position: "absolute",
+              top: "220px",
+              right: "460px",
+            }}
+          />
+        </Box>
+        <Box
+          sx={{
+            padding: "152px 300px 100px",
+            maxWidth: "1920px",
+            margin: "0 auto",
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+          }}
+          id="about"
+        >
+          <Typography
+            variant="h3"
+            sx={{
+              padding: "0 0 100px",
+              color: "#FFC08B",
+              textAlign: "center",
+              fontFamily: "'Press Start 2P', cursive",
+            }}
+          >
+            Об игре
+          </Typography>
+          <Typography
+            variant="h5"
+            sx={{
+              padding: "0 0 60px",
+              color: "#104603",
+              textAlign: "left",
+              fontFamily: "'Press Start 2P', cursive",
+            }}
+          >
+            Классический тавер дефенс. Цель игры - остановить монстров от
+            прохода через карту, путем постройки башен. Башни строятся за
+            деньги, которые игрок зарабатывает убийством монстров.
+          </Typography>
+          <Typography
+            variant="h4"
+            sx={{
+              padding: "0 0 60px 128px",
+              color: "#104603",
+              textAlign: "left",
+              fontFamily: "'Press Start 2P', cursive",
+            }}
+          >
+            Есть несколько типов башен:
+          </Typography>
+          {gameDetails.towers.map((tower) => (
+            <ItemComponent
+              key={Math.random()}
```

Comment:
Какая гадость. Не надо так делать )

```suggestion
              key={crypto.randomUUID()}
```

Reply:
А вообще во время рендера генерировать айдишник - это бесполезная срань. Он каждый раз всё равно будет новый. Тогда уж лучше использовать `tower.image` в качестве уникальный ключей